server/heroku/logs.go

```diff
@@ -24,9 +25,10 @@ func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter,
 	}
 
 	var form PostLogsForm
-
 	if err := Decode(r, &form); err != nil {
-		return err
+		if err.Error() != "EOF" {
```

Comment:
You can change this to:

```go
if err != io.EOF
```

I'd add a comment there as well just to mention why we're ignoring io.EOF.